include/glow/Base/Type.h

```diff
@@ -220,7 +220,6 @@ struct ShapeHWD {
     width = shape[1];
     depth = shape[2];
   }
-
```

Comment:
Nit, revert the loss of vertical whitespace :-)
(It's like an old SAT analogy, Bert:Vertical whitespace::Nadav:Periods at end of sentences)

Reply:
Huh thats weird I didn't do that on purpose